test/util/config/directory.go

```diff
@@ -0,0 +1,34 @@
+package config
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+)
+
+// FindDirectory returns the absolute path to a directory containing name within
+// the directory hierarchy of the caller. The search starts at the current working
+// directory of the caller
+func FindDirectory(name string) (string, error) {
```

Comment:
If we are planning on using this in production code, it should have unit tests.

Reply:
I dont think we gonna use it in prod?

Reply:
also shouldn't this be in fakerp/config?

what about using something like this:
https://www.reddit.com/r/golang/comments/6ru4fw/finding_the_projects_root_directory/

func GetSecretsDir() string {
confDir := importPathToDir("github.com/openshift/openshift-azure/pkg/config") 
return path.Join(confDir, "..","..","secrets")
}